docs/Server/API/V1/contents.md

```diff
@@ -154,6 +154,6 @@ URI returning the linked file's byte stream.
     "size": 18,
     "mode": "0o644",
```

Comment:
Given the recent discussions in #3396, should we be advertising `"mtime"` and `"mode"` here?

Reply:
It's all placeholder until we release 1.0, and it's all subject to change. I think for now I'll stick with "what's there" and when we've implemented the new `inventory` and `contents` we'll update.